app.js

```diff
@@ -114,6 +101,30 @@ const startApp = async () => {
       console.log('Something went wrong... Try again');
     }
   }
+
+  switch (options.watermarkType) {
+    case 'Text watermark':
+      const text = await inquirer.prompt([
+        {
+          name: 'value',
+          type: 'input',
+          message: 'Type your watermark text:',
+        },
+      ]);
+      putTextOnImage(text);
+
+      break;
+    default:
```

Comment:
Tutaj możesz wstawić case dla image zamiast default. Albo gdyby się rzucał linter to może byc tak np: 
```js
case "image":
default:
```

Reply:
Done.